src/components/__stories__/buttons.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import { storiesOf, action } from "@kadira/storybook"
+
+import Button, { ButtonState } from '../buttons/default'
+import GhostButton from '../buttons/ghost'
+import InvertedButton from '../buttons/inverted'
+
+storiesOf("Buttons", Button)
+    .add("Default Button", () => {
+        return (
+            <div>
+                <div>
+                    <Button>Button</Button>
+                    <Button disabled>Button</Button>
+                </div>
+                <div>
+                    <Button state={ButtonState.Success}>Button</Button>
+                    <Button state={ButtonState.Success} disabled>Button</Button>
+                </div>
+                <div>
+                    <Button state={ButtonState.Failure}>Button</Button>
+                    <Button state={ButtonState.Failure} disabled>Button</Button>
```

Comment:
The only thing I'd like to avoid is the need to import `{ ButtonState }`. Having a type check to specify a button state seems a bit overkill to me. Other than that, I'm :+1: